### DISCUSSION TITLE
https://cucoders.dev/publicaciones/osmely/btcpay-bitcoin-te-libera-/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#55] [Page Comments]
# https://cucoders.dev/publicaciones/osmely/btcpay-bitcoin-te-libera-/

Un acercamiento a BTCBay como pasarela de pagos en Bitcoin lista para usar.

https://cucoders.dev/publicaciones/osmely/btcpay-bitcoin-te-libera-/

**Comment by manuelernestog** [comment]
Buenísimo esto bro. 🎉🎉

**Reply by osmely** [reply]
Se acabaron las excusas! jaja, el que quiera usar BTC pues más fácil imposible.

**Reply by manuelernestog** [reply]
Así mismo es 👏👏

**Comment by kevjetsky** [comment]
Muy bueno, hay q probar eso 😌😋

**Reply by osmely** [reply]
Cuando quiera!!, el proceso es simple, me das un email y le envío link de invitación. A partir de ahi le damos 1 semana de pruebas a su cuenta :) y todos felices 😁😁😁

**Comment by RasorDev** [comment]
Sigue activo el servicio?

**Reply by osmely** [reply]
Desafortunadamente no, no tuvo el numero de usuarios requeridos
